frontend/src/components/App/Home/index.tsx

```diff
@@ -236,6 +236,24 @@ function HomeComponent(props: HomeComponentProps) {
       .sort();
   }
 
+  /**
+   * Gets the origin of a cluster.
+   *
+   * @param cluster
+   * @returns A description of where the cluster is picked up from: dynamic, in-cluster, or from a kubeconfig file.
+   */
+  function getOrigin(cluster: Cluster): string {
+    if (cluster.meta_data?.source === 'kubeconfig') {
+      const kubeconfigPath = process.env.KUBECONFIG ?? '~/.kube/config';
+      return t('translation|Kubeconfig') + ` (${kubeconfigPath})`;
```

Comment:
We don't need to translate Kubeconfig. And the value here should be: `kubeconfig:PATH_TO_FILE`
Can you make another PR to fix this, since this one is merged?

Reply:
This change as well as the other one mentioned are done here: #2543